src/api/routes/offer.js

```diff
@@ -226,4 +226,29 @@ export default (app) => {
                 return next(err);
             }
         });
+
+    /*
+     * Archives a given offer
+     *
+     * This action is irreversible
```

Comment:
It shouldn't. Eventually we may have an de-archive option

Reply:
Then what would be the difference between archiving and, say, hiding offers? It would just be a different name for the exact same thing. What the team agreed on was that archiving should be final, but if you know of a situation where de-archiving offers makes sense I'm more than happy to change this behavior.

Reply:
I know we discussed this in a meeting, but there's a difference even if we can de-archive them:
- Archived offers shouldn't be editable (I think)
- Archived offers don't appear in the `My Offers` company page
- Archived offers don't appear in the search results, even for admins

Still, I think it makes more sense to use this as a permanent action. If the company wants to use that offer again, imo we should have a way to replicate it in a new offer. What are your thoughts?

Reply:
Aside from this discussion, I have no more changes to ask

Reply:
Merged for now, if in the future we wish to make this reversible it will be done in a new PR.